package/Dockerfile

```diff
@@ -114,7 +114,7 @@ RUN curl -sLf ${!TINI_URL} > /usr/bin/tini && \
     mkdir -p /var/lib/rancher-data/driver-metadata
 
 ENV CATTLE_UI_VERSION 2.5.8-rc17
```

Comment:
This also needs to be changed?

Reply:
No its just a dashboard fix

Reply:
ok, looks weird. I thought these two are always bumped together.